.github/workflows/bot-coderabbit-plan-trigger.yml

```
@@ -28,8 +18,7 @@ jobs:
       cancel-in-progress: false
     # Only run when the newly added label is beginner, intermediate, or advanced (case-insensitive)
     if: >
```

actionlint (1.7.12)

[error] parser did not reach end of input after parsing the expression. 1 remaining token(s) in the input: ")"

(expression)